pkgs/by-name/sc/schroot/debian-patches.nix

```diff
@@ -0,0 +1,70 @@
+# Generated by debian-patches.sh from debian-patches.txt
+let
+  prefix = "https://salsa.debian.org/debian/schroot/-/raw/debian/master/1.6.13-4/debian/patches";
```

Comment:
This should use a specific commit instead of `master`

Reply:
It is using the tag `debian/master/1.6.13-4` (it's not a branch name)

Reply:
Interesting naming scheme. Then it is okay.